### DISCUSSION TITLE
What should our code of conduct include?

### DISCUSSION THREAD

**Discussion by glortiz** [Code4PuertoRico/horas#35] [Q&A] [answered]
- What would be considered proper use and behavior within the platform from both mentees and mentors? 
- Relating to another discussion topic, should mentors be able to charge for their services?
- Should users be able to report other users who are not following the guidelines? ie, charging for services or making inappropriate advances.

**Comment by glortiz** [comment]
On June 8's Hacknight it was decided that we would search for examples of codes of conduct and comunity guidelines that could be used in 1hora.org

Notes: https://docs.google.com/document/d/1F3R_TWgaFJuje73LEmXIoN7-jOhL4sZocI6v6UVb8c8/edit

**Reply by glortiz** [reply]
Startup Communities by Brad Feld (fotos) - https://drive.google.com/drive/folders/1yhDH64QV2giRF-DTJySevCr3lSbToxeW?usp=sharing

SCORE - https://www.score.org/volunteer

MENTOR - https://www.mentoring.org/take-action/become-a-mentor/

**Comment by m3orange** [comment, marked as answer]
Gian, there's good stuff in that text. I particularly liked the Techstars mentor manifesto (the Google Drive link), and I think it would be good to reference it, with proper credit, when we draft our code of conduct. I found it straightforward and insightful, with points like:

• Clearly separate opinion from fact.
• Hold information in confidence.
• Expect nothing in return (you’ll be delighted with what you do get back). 
• _**This one, particularly:**_ Know what you don’t know. Say I don’t know when you don’t know. “I don’t know” is preferable to bravado. 

I managed to find the original content: https://www.techstars.com/the-line/advice/mentor-manifesto

**Reply by glortiz** [reply]
I agree with giving proper credit, thanks for the feedback.

**Comment by glortiz** [comment]
Draft of:
Code of conduct
Mentor Role
Mentee Role

https://docs.google.com/document/d/1S2yVZ97byp3JgIV73u680OHKiJn1g0hVyWgR-rB03RQ/edit#
